Components/Grid/grid.css

```diff
@@ -0,0 +1,72 @@
+.grid-container {
+    display: flex;
+    flex-direction: column;
+    gap: var(--container-gap);
+    background-color: var(--box-color);
+    width: var(--container-width);
+    height: 15%;
+    justify-content: center;
+    align-items: center;
+    border-radius: var(--container-border-radius);
+    margin: var(--container-margin);
+}
+
+.grid-section{
+    display: grid;
+    width: 500px;
```

Comment:
Change this px into rem here since it's a big px value.

Reply:
Done!